pontos/nvd/cve_change_history/api.py

```diff
@@ -0,0 +1,164 @@
+                limit (without an API key) is 5 requests in a rolling 30 second
+                window. The rate limit with an API key is 50 requests in a
+                rolling 30 second window.
+                See https://nvd.nist.gov/developers/start-here#divRateLimits
+                Default: True.
+        """
+        super().__init__(
+            DEFAULT_NIST_NVD_CVE_HISTORY_URL,
+            token=token,
+            timeout=timeout,
+            rate_limit=rate_limit,
+        )
+
+    async def cve_changes(
+        self,
+        *,
+        change_start_date: Optional[datetime] = None,
+        change_end_date: Optional[datetime] = None,
+        cve_id: Optional[str] = None,
+        event_name: Optional[EventName] = None,
+    ) -> AsyncIterator[CVEChange]:
+        """
+        Get all CVEs for the provided arguments
+
+        https://nvd.nist.gov/developers/vulnerabilities#divGetCves
+
+        Args:
+            change_start_date: Return all CVE changes after this date.
+            change_end_date: Return all CVE changes before this date.
+            cve_id: Return all CVE changes for this Common Vulnerabilities and Exposures identifier.
+            event_name: Return all CVE changes with this event name.
+
+
+        Returns:
+            An async iterator to iterate over CVEChange model instances
+
+        Example:
+            .. code-block:: python
+
+                from pontos.nvd.cve_change_history import CVEChangeHistoryApi
+
+                async with CVEChangeHistoryApi() as api:
+                    async for cve_change in api.cve_changes(event_name=EventName.INITIAL_ANALYSIS):
+                        print(cve_change)
+        """
+        total_results: Optional[int] = None
+
+        if bool(change_start_date) ^ bool(change_end_date):
+            raise PontosError(
+                "change_start_date and change_end_date must be provided mutally"
+            )
+
+        params: Params = {}
+        if change_start_date and change_end_date:
+            if change_end_date - change_start_date > timedelta(days=120):
+                raise PontosError(
+                    "change_start_date and change_end_date must not be more than 120 days apart"
+                )
+
+            params["changeStartDate"] = format_date(change_start_date)
+            params["changeEndDate"] = format_date(change_end_date)
+
+        if cve_id:
+            params["cveId"] = cve_id
+
+        if event_name:
+            params["eventName"] = event_name
+
+        start_index: int = 0
+        results_per_page = None
+
+        while total_results is None or start_index < total_results:
+            params["startIndex"] = start_index
+
+            if results_per_page is not None:
+                params["resultsPerPage"] = results_per_page
+
+            response = await self._get(params=params)
+            response.raise_for_status()
+
+            data: Dict[str, Union[int, str, Dict[str, Any]]] = response.json(
```

Comment:
Would be nice to use `JSON` here https://github.com/greenbone/pontos/blob/main/pontos/nvd/api.py#L35
```suggestion
            data: JSON = response.json(
```

Reply:
Done in https://github.com/greenbone/pontos/pull/920/commits/355bea613c0079a625392e931d15f0f68cb04c9c.